mteb/tasks/Clustering/multilingual/WikiClusteringP2P.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClustering, MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGUAGES = {
+    "da": ["dan-Latn"],
+    "lv": ["lav-Latn"],
+    "gv": ["glv-Latn"],
+    "sq": ["sqi-Latn"],
+}
```

Comment:
Should we expand this?

Reply:
We totally could - if you have recommendations for languages I can run them this evening:))

Reply:
https://github.com/embeddings-benchmark/mteb/issues/366

^ anything not covered